OpenTrons/opentrons-api
# fix(app): set modal height to auto for all pipette wizard flows

closes [RQA-2279](https://opentrons.atlassian.net/browse/RQA-2279)

# Overview


# Test Plan

- open a 96-channel pipette wizard flow
- resize desktop app window
- observe that modal does not expand
https://github.com/Opentrons/opentrons/assets/47604184/948a0372-5098-4354-a9a9-85feac32507d

# Changelog

- remove height prop in pipette wizard flow modal 

# Risk assessment
low

[RQA-2279]: https://opentrons.atlassian.net/browse/RQA-2279?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
